docs/operational_guide/replication_and_deployment_in_zones.md

```diff
@@ -10,6 +10,22 @@ A replication factor of at least 3 is highly recommended for any M3DB deployment
 
 M3DB will do its best to distribute shards evenly among the availability zones while still taking each individual node's weight into account, but if some of the availability zones have less available hosts than others then each host in that zone will be responsible for more shards than hosts in the other zones and will thus be subjected to heavier load.
 
+
+### Dangers of Replication Factor of 1 or 2
```

Comment:
nit: i'd probably rephrase this to read a little better:

> Dangers of Using Bad Replication Factors

or

> Dangers of Using Replication Factor 1 or 2

or

> Dangers of Using Fewer Than 3 Replicas, or Using Even Numbers of Replicas

or

> Replication Factor Recommendations

depending on preference.